components/new_channel_modal/new_channel_modal.tsx

```diff
@@ -108,12 +97,11 @@ const NewChannelModal = () => {
     const [serverError, setServerError] = useState('');
 
     // create a board along with the channel
+    const pluginsComponentsList = useSelector((state: GlobalState) => state.plugins.components);
+    const createBoardFromChannelPlugin = pluginsComponentsList?.CreateBoardFromTemplate;
```

Comment:
nit: any reason for not having a oneliner here? like:

```suggestion
    const createBoardFromChannelPlugin = useSelector((state: GlobalState) => state.plugins.components?.CreateBoardFromTemplate);
```

Reply:
not very special one but I had two reasons, first readability and second, easier to mock for tests ;)